frontend/src/employee-mobile-frontend/common/TopBar.tsx

```diff
@@ -68,40 +74,87 @@ export default React.memo(function TopBar({
 }: Props) {
   const { i18n } = useTranslation()
 
+  const notificationResult = useQueryResult(currentSystemNotificationQuery())
+
+  // only shown on main screen
+  const hideSystemNotification = invertedColors
```

Comment:
I think the notification gets refetched basically on every page change, because TopBar is remounted on pretty much every page and the hook is unaware when we're not going to show the data. :thinking:

Reply:
for some reason it doesn't seem to, but I'll add the query enabled option to be sure